src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/data/UserSummaryWithEmail.kt

```diff
@@ -16,6 +16,8 @@ data class UserSummaryWithEmail(
   val active: Boolean,
   @Schema(description = "Caseload that is currently active, typically the prison the user is currently working at")
   val activeCaseload: PrisonCaseload?,
+  @Schema(description = "The count of DPS roles allocated to this staff member", example = "12")
+  val dpsRoleCount: Int,
```

Comment:
dpsRoleCount was needed to ensure returning the UserSummaryWithEmail returned all fields in UserSummary (the previously used return type)